frontend/packages/kubevirt-plugin/integration-tests/tests/models/importWizard.ts

```diff
@@ -50,36 +60,88 @@ export class ImportWizard extends Wizard {
     await selectOptionByText(view.vcenterInstanceSelect, instance);
   }
 
-  async fillHostname(hostname: string) {
+  async vmwareFillHostname(hostname: string) {
     await fillInput(view.vcenterHostnameInput, hostname);
   }
 
-  async fillUsername(username: string) {
+  async vmwareFillUsername(username: string) {
     await fillInput(view.usernameInput, username);
   }
 
-  async fillPassword(password: string) {
+  async vmwareFillPassword(password: string) {
     await fillInput(view.vcenterPasswordInput, password);
   }
 
+  async rhvFillApi(apiUrl: string) {
```

Comment:
I think that having separate methods for rhv and vmware for all these fields is very difficult to read and orient in.
I think we should add data-test-id attributes that would be the same for certain field across provider types.
@suomiy what do you think?

Reply:
Sounds reasonable, the only objection is that I need to complete this tests ASAP and this kind of refactoring will take some time

Reply:
I think these changes should not be that intrusive and would be nice to have for future readability. Btw is there some reason/deadline for this urgency?

Reply:
Done, I've implemented a separate class rhvImportWizard

Reply:
+1, could you please also make a VMwareImportWizard class and leave the ImportWizard class to share common functionality?